app/views/auth.html

```diff
@@ -12,8 +12,9 @@
 
     <div class="loginFields">
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
+      <br>
```

Comment:
We also have to make sure it looks good when the user wants to log in as well for registering.